lib/worker.js

```diff
@@ -36,7 +36,12 @@ process.on('message', ({ filename, fix, id, text }) => {
   }
 
   try {
-    linter.lintText(text, { filename, fix }, (err, { results } = {}) => {
+    /* In standard-engine <6.0.0 the linter.lintText was async, but from
+     * version 6.0.0 it changed to be sync. This callback is left in here to
+     * handle the older versions of standard-engine based linters. We can
+     * remove it in a later major version, if it becomes a liability.
+     */
+    const returnValue = linter.lintText(text, { filename, fix }, (err, { results } = {}) => {
```

Comment:
I think the callback should check if `returnValue` is falsy, as an extra guard against strange `standard-engine` implementations. (I'm thinking here about implementations that follow `standard-engine`'s interface, not `standard-engine` itself.)

That said the calling code is promise based so repeated messages are ignored anyway.